src/api/SFA.DAS.Provider.Events.Api.IntegrationTests/app.config

```diff
@@ -4,14 +4,14 @@
     <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog, Version=4.0.0.0, Culture=neutral, PublicKeyToken=5120e14c03d0593c" />
   </configSections>
   <connectionStrings>
-    <add name="default" connectionString="Server=.;Database=ProviderEventsAT;Trusted_Connection=True;" />
+    <add name="default" connectionString="Server=(localdb)\ProjectsV13;Database=ProviderEventsAT;Trusted_Connection=True;" />
```

Comment:
You'll want to change these connection strings back otherwise tests won't run for everyone else.

Reply:
done